overlord/fdestate/backend/reseal.go

```diff
@@ -145,7 +155,10 @@ func doReseal(manager FDEStateManager, method device.SealingMethod, rootdir stri
 
 	var keys []resealParamsAndLocation
 
+	var devices []string
+
 	for _, disk := range disks {
+		devices = append(devices, disk.DevPath())
```

Comment:
Is maybe partition a better name than disk here? I found it a bit confusing. Also, maybe worth adding a comment saying that the key used for sealing data and save is in fact the same and that that is the only one we seal.

Reply:
"block" is would be better then. I would like to avoid saying "partition" since in theory it could be something else.

Reply:
Or "container".

Reply:
Renamed "container". Also added a comment.